homeassistant/components/generic/config_flow.py

```diff
@@ -126,27 +133,12 @@ def build_schema(
     return vol.Schema(spec)
 
 
-def get_image_type(image: bytes) -> str | None:
-    """Get the format of downloaded bytes that could be an image."""
-    fmt = None
-    imagefile = io.BytesIO(image)
-    with contextlib.suppress(PIL.UnidentifiedImageError):
-        img = PIL.Image.open(imagefile)
-        fmt = img.format.lower()
-
-    if fmt is None:
-        # if PIL can't figure it out, could be svg.
-        with contextlib.suppress(UnicodeDecodeError):
-            if image.decode("utf-8").lstrip().startswith("<svg"):
-                return "svg+xml"
-    return fmt
-
-
 async def async_test_still(
     hass: HomeAssistant, info: Mapping[str, Any]
 ) -> tuple[dict[str, str], str | None]:
     """Verify that the still image is valid before we create an entity."""
-    fmt = None
+    fmt: str | None = None
+    content_type: str | None
```

Comment:
Should there be a definite assignment here?
```suggestion
    content_type: str | None = None
```

Reply:
Agreed, done afae319